src/template_options.ts

```diff
@@ -71,6 +73,12 @@ export class TemplateOptions {
       throw new Error('Missing laguage value');
     }
     this.style = options.style ? options.style : null;
+
+    if(options.overWriteDir === 1) {

```

Comment:
please, check this via `overWriteDir` enum. Enum not binds to concrete values.

Reply:
Not needed anymore after other fixes.